### ISSUE TITLE
Suggestion: replace dropout with batch norm

### ISSUE THREAD

**Issue by tybrad11** [MrGiovanni/UNetPlusPlus#2]
I'm testing a small data set (training on 10 CT liver segmentations, testing on 4), but I get WAY better results when I replace dropout with batch norm in the standard_unit(). May be data size dependent, but worth noting.

**Comment by wl202305** [comment]
hello，now,i am look this project.because i am a beginner.can you tell
me the 'main' file of  this project?thank you very much.my email:wl202305@126.com

**Comment by MrGiovanni** [comment]
@wl202305 : We have updated the README. You can use 'BRATS2013_application.py' to run the code, also, you can use our code examples for your own data. Thanks!
